datalad/downloaders/tests/test_http.py

```diff
@@ -123,26 +126,46 @@ def enter_new(self):
         # pop last used credential, so we would use "new" ones
         del self._fixed_credentials[0]
 
+
```

Comment:
to this same branch (`nf-ui-auth-downloaders`).  i don't see it (and it says your last commit to that branch was 6 days ago on your fork (https://github.com/yarikoptic/datalad/tree/nf-ui-auth-downloaders)